packages/http-smoke-testing/src/RequestDataSetGenerator.php

```diff
@@ -27,6 +30,54 @@ public function __construct(RouteInfo $routeInfo)
         $this->routeInfo = $routeInfo;
         $this->defaultRequestDataSet = new RequestDataSet($this->routeInfo->getRouteName());
         $this->extraRequestDataSets = [];
+
+        if ($routeInfo->getAnnotations() !== null) {
```

Comment:
It is not good practice to run code from constructor

Reply:
Fixed